bin/bower

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env node
 
-var fs = require("fs");
+var fs = require("fs"),
+    pkg = require("../package.json");
```

Comment:
instead of require you may use `pkg = JSON.parse(path.resolve(__dirname, '..', 'package.json')); ` or something like that. Require not only parses but executes